.gitlab/custom-jobs-and-variables.yml

```diff
@@ -50,49 +55,17 @@ variables:
 # Arguments for job level allocation
 # Note: We repeat the reservation, necessary when jobs are manually re-triggered.
   RUBY_JOB_ALLOC: "--reservation=ci --overlap --nodes=1"
-# Project specific variants for ruby
-  PROJECT_RUBY_VARIANTS: ""
-# Project specific deps for ruby
-  PROJECT_RUBY_DEPS: ""
```

Comment:
Please do not remove those variables.

Reply:
They are not used anywhere, so I removed them to improve readability. Do you prefer to have them here to minimize the diffs with some template version of this file, maybe in the `radiuss/radiuss-shared-ci` repo?